Scripts/datahandling/zonedata.py

```diff
@@ -38,8 +38,8 @@ def __init__(self, data_dir, zone_numbers):
         transit_zone["dist_fare"] = transit_zone["fare"].pop("dist")
         transit_zone["start_fare"] = transit_zone["fare"].pop("start")
         self.transit_zone = transit_zone
-        car_cost = read_csv_file(data_dir, ".cco", squeeze=True)
-        self.car_dist_cost = car_cost[0]
+        car_cost = read_csv_file(data_dir, ".cco", squeeze=False)
+        self.car_dist_cost = car_cost["dist_cost"][0]
```

Comment:
This one worked before too, but ok then.

Reply:
The problem was that cco-files in test-data folder did not have headers and full model files in Sharepoint had. The files with header were read as an array with two values and index [0] returned the header string.

I thought that this would give more explicit error when reading zonedata, if header is not found. Otherwise the error would point to assignment, where model tries to multiply float with string.